klarna-on-demand/src/main/java/com/klarna/ondemand/RegistrationActivity.java

```diff
@@ -24,9 +30,19 @@ protected String getUrl() {
 
     @Override
     protected void handleUserReadyEvent(Map<Object, Object> payload) {
-        String token = (String) payload.get("userToken");
         Intent result = new Intent();
-        result.putExtra(EXTRA_USER_TOKEN, token);
+
+        // Backwards compatibility.
```

Comment:
This *does* allow us not to change the docs, but that is not a good around reason to hang on to this.